src/infrastructure/figma/transformers/figma-file-transformer.ts

```diff
@@ -31,6 +37,8 @@ export const transformFileToAtlassianDesign = ({
 		status: AtlassianDesignStatus.NONE,
 		type: AtlassianDesignType.FILE,
 		lastUpdated: fileResponse.lastModified,
-		updateSequenceNumber: new Date(fileResponse.lastModified).getTime(),
+		updateSequenceNumber: getUpdateSequenceNumberFrom(
```

Comment:
**note:** Figma `GET /v1/files/:key` and `GET /v1/files/:key/meta` return "Last Modified" with different precision. Update all the transformers to use the normalized value.

Please, see here fore more detail:
https://github.com/atlassian-labs/figma-for-jira/pull/151/files#diff-3472e15fcccb89a09c8f094d294a6c0c63511fcd6ba60e7f269ffaa5b1dcde95R82